modules/sdk-coin-atom/test/unit/transactionBuilder/StakingActivateBuilder.ts

```diff
@@ -0,0 +1,26 @@
+import { getBuilderFactory } from '../getBuilderFactory';
+import * as testData from '../../resources/atom';
+import should from 'should';
+import { toHex, TransactionType } from '@bitgo/sdk-core';
+import { Transaction as AtomTransaction } from '../../../src/lib/transaction';
+import { fromBase64 } from '@cosmjs/encoding';
+
+describe('Atom Delegate txn Builder', () => {
```

Comment:
please add more test, include "deserialize, add signature", see sui's test as a reference

Reply:
Added more tests and verifying more data from the result